onboard/catkin_ws/src/data_pub/scripts/imu.py

```diff
@@ -6,8 +6,11 @@
 import traceback
 
 from sensor_msgs.msg import Imu, MagneticField
-from tf.transformations import euler_from_quaternion, quaternion_from_euler
+from tf.transformations import euler_from_quaternion, quaternion_from_euler, quaternion_multiply 
+#from tf import Quaternion
 
+transformation_quat = (0.7315563328, 0.6817809566, 0.000243224431, 0)
+transformation_quat_conj = (0.7315563328, -0.6817809566, -0.000243224431, 0)
 
```

Comment:
Quaternion stuff was from debugging IMU issues. I assume this is all fixed and this version can be ignored

Reply:
Don't merge